killbill/payment_method.py

```diff
@@ -0,0 +1,59 @@
+#
+# Copyright 2018 Quentral S.r.l.
```

Comment:
Hi pierre, 
I've done all the changes you told me.
Please check if you need other changes or is it ok!

Reply:
Thanks! We'll take a look.